packages/integration-tests/projects/suite-web/tests/browser/edge.test.ts

```diff
@@ -1,4 +1,4 @@
-// @beta
+// @stable
```

Comment:
this actually should not be part of this branch, you have probably made something wrong with git (regarding merge or rebase) would be best to drop 63a979021195e965228f4ad0392289f098955e1c and b12a086268f6604c0451baa6a8bb2472c507682e. My favourite way to do this is to do local rebase, something like `git rebase -i HEAD~22` and then use drop for commits you dont want to have in your branch.

Reply:
I don't have these commits in the branch in the local branch 
<img width="928" alt="Screenshot 2020-06-18 at 17 38 09" src="https://user-images.githubusercontent.com/61698325/85044216-9d8d2700-b18d-11ea-8a59-1ee76003af9a.png">